web-ui/src/main/resources/catalog/views/default/less/gn_topics_default.less

```diff
@@ -159,9 +182,34 @@
   height: 100%;
   line-height: 100%;
   padding-left: 8px;
-  padding-right: 8px;
+  padding-right: 4px;
   margin-top: 13px;
   font-size: 13px;
+  max-width: 13vw;
   white-space: nowrap;
   color: #fff;
+}
+@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
```

Comment:
The problem was with truncating the label, that's not working well with precentages